src/test/java/uk/gov/companieshouse/api/testdata/service/impl/CompanyPscStatementServiceImplTest.java

```diff
@@ -71,7 +69,7 @@ void create() {
         assertNotNull(statement.getNotifiedOn());
         assertEquals(ETAG, statement.getEtag());
         assertEquals("persons-with-significant-control-statement", statement.getKind());
-        assertEquals("no-individual-or-entity-with-signficant-control", statement.getStatement());
+        assertEquals("psc-exists-but-not-identified", statement.getStatement()); // Default case
```

Comment:
psc-exists-but-not-identified could has been declared as a constant to enhance readability. It has been used in multiple tests.